### DISCUSSION TITLE
Locast just shutdown

### DISCUSSION THREAD

**Discussion by Thunder7ga** [tgorgdotcom/locast2plex#301] [General]
Well, the lawsuits have succeeded.  As of this moment, Locast is no longer functional.

**Comment by K3v1n-M** [comment]
It is a sad day.

**Comment by CTJohnK** [comment]
I wonder if they had a mass cancellation after the removal of 'nag' messages yesterday which forced this action.

**Reply by K3v1n-M** [reply]
I stopped my subscription because I did not know what direction they were heading.
